tests/common_for_tests.py

```diff
@@ -82,6 +82,7 @@ def sensor_bands_info():
 def multi_raster_16b():
     source_raster = make_test_raster(4200, ['green', 'red', 'nir', 'blue'], dtype=np.uint16)
     array = source_raster.image.data
+    array.setflags(write=1)
```

Comment:
I prefer not to because, I like it to be `(write=0)` when testing so if we try to write to `image` in the code it will fail in tests.
that is why I need to change it every time before preparing data test.